process/transaction/shardProcess.go

```diff
@@ -789,6 +789,20 @@ func (txProc *txProcessor) processInnerTx(
 
 	txFee := txProc.computeInnerTxFee(innerTx)
 
+	prevTxHash, err := txProc.getPrevTxHashForInnerTxOfRelayedV3(originalTxHash, innerTx)
```

Comment:
this is added without any flags and it might not be backwards compatible if an error occurs. Previously no error was returned here.

Reply:
the flag is inside `getPrevTxHashForInnerTxOfRelayedV3`.. if the flag is not active yet, it will simply return originalTxHash and nil, as it is currently on devnet.. thus it should work as intended

Reply:
ok